src/scala/microsoft-spark-3-0/src/main/scala/org/apache/spark/deploy/dotnet/DotnetRunner.scala

```diff
@@ -164,12 +167,18 @@ object DotnetRunner extends Logging {
     }
   }
 
-  private def validateSparkVersions: Unit = {
-    if (!supportedSparkVersions(SPARK_VERSION)) {
+  private def validateSparkVersions(conf: SparkConf): Unit = {
+    val ignorePatchVersion = conf.get(DOTNET_IGNORE_SPARK_PATCH_VERSION_CHECK)
+    val majorMinorVersion = supportedSparkMajorMinorVersion.mkString("", ".", ".")
```

Comment:
just store it as a string to avoid `mkString`?

Reply:
Sure.  Do you want me to hardcode it in this method or set it as a class field at the top of the class definition ?

Reply:
Class field seems fine to me.